Pkgfile

```diff
@@ -178,6 +178,11 @@ vars:
   libnl_sha256: 9fe43ccbeeea72c653bdcf8c93332583135cda46a79507bfd0a483bb57f65939
   libnl_sha512: 80fbbc079299c90afd2a5eda62e4d4f98bf4ef23958c3ce5101f4ed4d81d783af733213bb3bab15f218555d8460bc2394898f909f4ac024fc27281faec86a041
 
+  # renovate: datasource=git-tags depName=SELinuxProject/selinux
```

Comment:
i guess we can just define one selinux version, since we re-use the same tarball everywhere? :thinking:

Reply:
Not the tarball, but tag. Perhaps will do so

Reply:
while at this, also could you update this back to just https://github.com/siderolabs/tools/blob/main/.github/renovate.json#L40 `SELinuxProject/selinux`

Reply:
Are hashes handled by us, not Renovate? Is Renovate just for notification purposes? Since if I make hash names different from the version name it might become unhappy

Reply:
renovate is just notification, we still handle hashes, one day we can handle it ourselves